templates/monitoring-bastion-template.yaml

```diff
@@ -828,7 +828,7 @@ Resources:
                 - BannerUrl: !If
                   - DefaultBanner
                   - !Sub
-                    - s3://${S3Bucket}/${QSS3KeyPrefix}scripts/banner_message.txt
+                    - s3://${S3Bucket}/${QSS3KeyPrefix}submodules/quickstart-linux-bastion/scripts/banner_message.txt
                     - S3Bucket: !If [ UsingDefaultBucket, !Sub 'aws-quickstart-${AWS::Region}', !Ref 'QSS3BucketName' ]
```

Comment:
This we may want to customize show users know they are on the monitoring host rather than the actual Bastion host.

Reply:
Added custom banner now